src/applications/find-va-forms/actions/index.js

```diff
@@ -76,6 +77,10 @@ export const fetchFormsThunk = (query, options = {}) => async dispatch => {
     dispatch(fetchFormsSuccess(results));
   } catch (error) {
     // If we are here, the API request failed.
-    dispatch(fetchFormsFailure());
+    dispatch(
+      fetchFormsFailure(
+        'Something went wrong. Our servers may be offline. Please try again later.',
```

Comment:
I think we should stick to our generic error message - `We’re sorry. Something went wrong on our end. Please try again later.`.  We have a sweet messaging guide somewhere, but I have no idea where it is anymore after the repo reorganization.

Reply:
do we already track these errors in prometheus?

Reply:
Sounds great! Will update it now :)

Reply:
If these errors are already tracked in Prometheus (not https://en.wikipedia.org/wiki/Prometheus btw lol), then i think we're good and probably don't need to also log it in Sentry!